src/Microsoft.Spatial/Microsoft.Spatial.csproj

```diff
@@ -163,6 +163,7 @@
       <DependentUpon>Microsoft.Spatial.tt</DependentUpon>
       <ExcludeFromStyleCop>true</ExcludeFromStyleCop>
     </Compile>
+    <None Include="packages.config" />

```

Comment:
do you change anything for Microsoft.Spatial project? I think no, right? please revert them

Reply:
I had to update packages to get it to build, but if others aren't having the problem I'll revert.

---
In reply to: [340910638](https://github.com/OData/odata.net/pull/1583#discussion_r340910638) [](ancestors = 340910638)